src/main/fc/init.c

```diff
@@ -828,7 +826,9 @@ void init(void)
 #ifdef USE_BARO
     baroStartCalibration();
 #endif
+
     positionInit();
+    positionControlInit();
```

Comment:
```suggestion
    positionControlInit(positionControlConfig());
```

Reply:
Thanks for these more sophisticated ways of doing these things.  Much appreciated.